### PR TITLE
GitHub #22701: Adding entry for etcd operator

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://github.com/openshift/openshift-docs/issues/22701

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@hexfusion @retroflexer Would you mind reviewing this entry for the etcd operator to our list of operators? To make sure that I got all of the items correct. And let me know if there is anything in particular that we need to note special about it (you can see that some operators have a "Notes" section).

Preview: https://github-22701--ocpdocs.netlify.app/openshift-enterprise/latest/operators/operator-reference.html#etcd-cluster-operator_red-hat-operators

**Review by kalexand-rh — COMMENTED** [review]
LGTM

(I think that adding some context to the command in the Configuration objects section might be helpful, but this change fits with the format of the existing Operator reference.)

File: modules/etcd-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//
+// * operators/operator-reference.adoc
+
+[id="etcd-cluster-operator_{context}"]
+= etcd cluster Operator
+
+[discrete]
+== Purpose
+
+The etcd cluster Operator handles the scaling of etcd and provisioning etcd dependencies such as TLS certificates. The etcd cluster Operator simplifies the disaster recovery procedure to restore to a previous cluster state, automates the addition of etcd members, provides more accurate etcd member health reporting, and reports events to assist with debugging the etcd cluster.
```
Comment (ironcladlou) [inline modules/etcd-operator.adoc:11]: Throughout: Odd capitalization of Operator and inconsistent naming — let's choose a consistent way to refer to the component, e.g. "the etcd operator".

**Review by ironcladlou — CHANGES_REQUESTED** [review]

File: modules/etcd-operator.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/etcd-operator.adoc:11]: @ironcladlou right now docs follows guidance to capitalize all instances of "Operator". I'm not personally a fan, but it's what we follow. But it's consistent with how we refer to it elsewhere in the docs, for example here: https://docs.openshift.com/container-platform/4.5/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

Let me know if you see anywhere in the docs that it's called something other than "etcd cluster Operator".

Comment (bergerhoffer) [reply modules/etcd-operator.adoc:11]: @ironcladlou But other than that ^, does the rest of the info in this look correct?

**Comment by bergerhoffer** [comment]
@geliu2016 Can you review this entry to add the etcd cluster operator to our list of shipped operators?

Preview: https://github-22701--ocpdocs.netlify.app/openshift-enterprise/latest/operators/operator-reference.html#etcd-cluster-operator_red-hat-operators

File: modules/etcd-operator.adoc (hunk 1 above)
Comment (ironcladlou) [reply modules/etcd-operator.adoc:11]: >The etcd cluster Operator simplifies the disaster recovery procedure to restore to a previous cluster state, automates the addition of etcd members, provides more accurate etcd member health reporting, and reports events to assist with debugging the etcd cluster.

I would suggest a few simplifications for consideration:

>The etcd cluster Operator automates etcd cluster scaling, enables etcd monitoring and metrics, and simplifies disaster recovery procedures.

Comment (bergerhoffer) [reply modules/etcd-operator.adoc:11]: Done, thanks!

**Comment by geliu2016** [comment]
LGTM, thanks

**Pull request merged**